.github/workflows/deploy.yml

```diff
@@ -34,10 +34,6 @@ jobs:
         run: sudo npm i yalc -g
```

Comment:
Just notice there is yalc, can we also remove this?
```
name: Install Yalc
run: sudo npm i yalc -g

Reply:
Yep